README.md

```diff
@@ -55,8 +55,8 @@ Quickstart
 - Create a playbook file with that content, e.g. `your_playbook.yml`
 - Fetch this role `ansible-galaxy install -n -p ./roles Graylog2.graylog-ansible-role`
 - Install role's dependencies `ansible-galaxy install -r roles/Graylog2.graylog-ansible-role/requirements.yml -p ./roles`
-- Apply the playbook to a Vagrant box `ansible-playbook your_playbook.yml -i "127.0.0.1:2222,"`
-- Login to Graylog by opening `http://127.0.0.1:9000` in your browser. Default username and password is `admin`
+- Run the playbook with `ansible-playbook your_playbook.yml -i "<host IP>,"`
```

Comment:
you mean why I don't keep the IP and port that Vagrant would give you _if_ you would use Vagrant and _if_ it would be the first box to be started?

I don't like vendor locking very much; next thing we might do is to attach the Vagrantfile in this repo and although Vagrant is _one_ way of testing this role, it's _not_ the only one.

Besides, if people can't resolve that on their own, it's very likely that they're missing some fundamentals without which, I must say, they're soon going to be lost in this journey 😄

but of course, feel free to change as you please.